docs/reference/snap-build-process.rst

```diff
@@ -0,0 +1,131 @@
+.. _snap-build-process:
+
+Snap build process
+==================
+
+This page describes at a high level the workflow between a snap recipe and its
+resulting artefact.
+
+
+The recipe file
+---------------
+
+Snaps are created using a build configuration -- called from here on a *recipe*
```

Comment:
Opinion - you could mention snap recipes are declarative as this is a fundamental design of snap recipes that is not seen in many other packaging tools.

Reply:
1df1efa25